pre-docs/writing_tasks.md

```diff
@@ -276,6 +276,48 @@ The following features are defined by default:
 * `shell`: present if the target has a posix shell
 * `powershell`: present if the target has powershell
 
+### Tasks with multiple files
```

Comment:
Isn't this "multiple tasks with one file?

I don't see how require_relative worksbecause the other file won't be available on the target?
Where does the `run_task` method below come from?

This feature is more about having one init.rb task and using it in the `implementations` section of `hello.json` and `goodbye.json`